common/environment/environment.cpp

```diff
@@ -966,12 +1004,42 @@ void CLocalEnvironment::buildMachineCache() const
                 x.append(name).append("\"]");
                 Owned<IConstEnvBase> cached = new CConstMachineInfo((CLocalEnvironment *) this, &it->query());
                 cache.setValue(x.str(), cached);
+                if (streq(".", name) || strieq("localhost", name) || streq("127.0.0.1", name) || streq(name, localHostIP.str()))
```

Comment:
Yes.
For some reason I am not receiving email notifications from github for this PR (and perhaps others).